buildroot-external/package/generic_raw_uart/0002-Linux-6.12-llseek.patch

```diff
@@ -0,0 +1,16 @@
+Remove llseek NULL'd in 6.0 (868941b) removed in 6.12 (cb787f4)
+
+Signed-off-by: Nick Venenga <nick@venenga.com>
```

Comment:
Please compare to the other *.patch files again. You patch files are still missing an `Upstream:` statement which otherwise would end up in the buildroot linter check to fail.

Reply:
I assume you mean like `board/hardkernel/odroid-m1/patches/uboot/0001-odroid-m1-Disable-eMMC-DDR52-mode-enable-HS200.patch` described in https://buildroot.org/downloads/manual/patch-policy.txt#:~:text=Additional%20patch%20documentation

I see 119 patches but only 3 have `Upstream:`

Additionally, is there a way to run this linter locally to verify all requirements are satisfied?

Reply:
@jens-maus Actually @nijave is kind of correct here - HAOS has many patches without `Upstream` tag and its checking is disabled in CI: <https://github.com/home-assistant/operating-system/blob/dev/.github/workflows/pr-checks.yml#L34>